core/templates/components/main-nav/main-nav.twig

```diff
@@ -0,0 +1,388 @@
+
+  {% if has_search == true %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav su-main-nav--mobile-search {{ modifier_class }}" aria-label="Main">
+  {% else %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav {{ modifier_class }}" aria-label="Main">
+  {% endif %}
+  <button id="su-main-nav__toggle" class="su-main-nav__toggle {{ button_modifier_class }}" aria-expanded="false">Menu</button>
```

Comment:
We have a button component can we not use that here? Also, please no IDs in components as they can be used more than once on a page.